sources/web/datalab/polymer/components/file-browser/file-browser.html

```diff
@@ -102,6 +102,7 @@
         display: flex;
         flex-direction: column;
         flex: 1 1 auto;
```

Comment:
Do we need this line if we have a 100% height?

Reply:
Apparently not. Removed.